core/src/stores/migrations/20241210_nodes_parent.sql

```diff
@@ -0,0 +1,4 @@
+-- data sources nodes
+ALTER TABLE data_sources_nodes ADD COLUMN parent_id TEXT;
```

Comment:
Wait I thought we were not materializing them?

Reply:
sorry, forgot to remove it